python/lsst/ts/m2com/mock/mock_control_closed_loop.py

```diff
@@ -260,8 +259,45 @@ def load_file_lut(self, filepath):
                 self._lut[name] = np.float64(data)
                 if name == "F_E":
                     self._lut["lutInAngle"] = np.float64(data.keys())
-            else:
-                self._lut[name] = np.loadtxt(filepath / lut)
+
+        self._lut["temp_inv"] = self._calc_temp_inv_matrix()
+
+    def _calc_temp_inv_matrix(self):
```

Comment:
it doesn't look like this method need to be a member of this class, e.g., it does not access any class attribute... Consider either extracting this to a submodule (utils?) or making it static.. I think the first is probably better since it keeps the class interface cleaner..

Reply:
Put the function into the static method in commit: [c313c57](https://github.com/lsst-ts/ts_m2com/pull/14/commits/c313c57ab18054e2f5878d1c68d5dcd545f4ae34). I think this function is more specific to the closed-loop control. Thanks!